src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -316,6 +319,9 @@ std::map<std::string, ov::Any> properties_to_any_map(const std::map<std::string,
             };
             ov::EncryptionCallbacks encryption_callbacks{encrypt_func, decrypt_func};
             properties_to_cpp[property.first] = encryption_callbacks;
+        } else if (property.first == ov::hint::model.name()) {
+            auto model = Common::utils::convert_to_model(property.second);
+            properties_to_cpp[property.first] = std::static_pointer_cast<const ov::Model>(model);
```

Comment:
Is there any tests to catch this issue?

Reply:
actually no, ov.hint.model is device specific property (only NPU and GPU have it), and python api tests are device agnostic. Just manual testing right now.